instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java

```diff
@@ -72,6 +84,22 @@ public Builder remoteServiceName(String remoteServiceName) {
       return this;
     }
 
+    /**
+     * Opt-out of sharing poll span when no trace-context or extra propagation field is available on
```

Comment:
Since the flag defaults to true, this isn't really an opt-out. Can just say `Controls the sharing of a poll span for incoming spans with no trace context`.